cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/constraint/TimeRangeConstraintTest.java

```diff
@@ -77,7 +77,11 @@ public void testForwardRange() {
     // 3:24PM PST
     long now = 1494368640000L;
 
-    SimpleJob job = new SimpleJob(SCHEDULE, now, Collections.<Notification>emptyList(), Job.State.PENDING_TRIGGER);
+    ProgramSchedule schedule = new ProgramSchedule("SCHED1", "one partition schedule", WORKFLOW_ID,
```

Comment:
not sure, this is not my change. Must have come in through rebase+conflict resolution. Fixed now.